app/controllers/questions/at_capacity_controller.rb

```diff
@@ -3,6 +3,12 @@ class AtCapacityController < QuestionsController
     include AnonymousIntakeConcern
     layout "application"
 
+    def self.show?(intake)
+      return true if intake.client.routing_method_at_capacity?
+
+      intake.vita_partner&.at_capacity?
```

Comment:
the old at capacity page routing was a bit weird, because it seemed to let through the people who did not have a vita partner assigned, but we actually will not assign a vita partner when we're at capacity....

I'm considering moving the second line intake.vita_partner.at_capacity? from here as well, because we shouldnt assign them to that vita partner on the previous page until there IS capacity. Does that seem right?